docs/templates/how-to.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Add a title (using an infinitive like this one)
+sidebarTitle: Optional short title if 'title' is long
+description: A short description (up to ~160 characters) of the article that should make sense out of context (like on a listing page).
+---
+
+<!-- 
+When to use
+  When there is a single outcome a user wants to archieve.
+  When you want to explain how to get to the outcome in ordered steps.
+  https://diataxis.fr/how-to-guides/ 
+
+How to use
+  1. Copy this template into the right directory in /src/docs/.
+  2. Rename it to match the title.
+  3. Replace the following content with your own.
+-->
+
+Introduce the topic in 1 or 2 sentences.
+Include reasons why a person might want to do it
+(the most common use cases).
+Also include what the goal of the task is.
+
+## Before you begin
+
+Include here any terms the reader might need
+(either defined here or linked to a definition).
+Also any prerequisites they might need to install/know,
+such as having the CLI installed or having created a project.
+For tasks like installing the CLI,
+link to instructions rather than including them here.
+
+## 1. Do this first
+
+This starts the list of steps readers must do in order.
+Order them with numbers.
+The titles should be bare infinitives: https://developers.google.com/style/headings?hl=en
+
+Introduce the purpose of this step very briefly.
+
+Then use an ordered list for the actions to take:
+
+1. Do this first.
+1. Do this next.
+1. Do this third.
+1. Do this fourth.
+
+Include a short example of what the expected outcome of this step is,
+such as a code example, a description of what the user can do now,
+or (if absolutely necessary) a screenshot.
+
+## 2. Do this second
+
+Split into separate steps when there are more than 5--7 actions in a step.
```

Comment:
I think external contributors might not understand at first what we mean by split into separate steps. It could be creating 2 different tasks, using substeps or something else. If they don't have experience creating how to content, they might get confused.  I've taken the liberty to add an alternative explanation. 

"Before you start writing, think about the number of steps a task might have. If it has more than 5-7 steps, you might want to consider breaking the tasks into two separate tasks or using substeps. 
Also, split the steps when each set has a different outcome"

Reply:
I totally understand the potential for confusion. I was intentionally trying to avoid terms like "substeps" because I don't think they really add clarity. Here, "steps" are the sections and "actions" are the items in the list. I can make that more clear, especially since I didn't use that in step 1 :facepalm: .